resolvers/node/resolution.rs

```diff
@@ -54,7 +55,7 @@ pub static DEFAULT_CONDITIONS: &[&str] = &["deno", "node", "import"];
 pub static REQUIRE_CONDITIONS: &[&str] = &["require", "node"];
 static TYPES_ONLY_CONDITIONS: &[&str] = &["types"];
 
-fn conditions_from_resolution_mode(
+pub fn deno_conditions_from_resolution_mode(
```

Comment:
Perhaps this can instead be a method on the `ResolutionMode` enum?

```
impl ResolutionMode {
  pub fn default_conditions(&self) -> &'static [&'static str] {
    // ...
  }
}

Reply:
Done